src/__tests__/test-cli-args.ts

```diff
@@ -40,3 +40,21 @@ test("handles debug flag", () => {
   expect(parsedArgs.inputDirectory).toBe(".");
   expect(parsedArgs.debug).toBe(true);
 });
+
+describe("Handle json flag", () => {
+  test("option --json", () => {
+    const inputArgs = [".", "--json"];
+
+    const parsedArgs: IArgs = parseInputParameters(inputArgs);
+
+    expect(parsedArgs.json).toBeTruthy();
+  });
+
+  test("option -j", () => {
```

Comment:
I wound't have a short flag for the `json` output. We don't do this for the same flag on the main `snyk` command. `--json` is short enough and explicit.